crates/holochain/src/core/queue_consumer/publish_dht_ops_consumer.rs

```diff
@@ -15,11 +15,26 @@ pub fn spawn_publish_dht_ops_consumer(
     conductor: ConductorHandle,
     network: impl HolochainP2pDnaT + Clone + 'static,
 ) -> TriggerSender {
+    #[cfg(feature = "test_utils")]
```

Comment:
I don't want to expose configuring this outside of testing. Sure somebody could modify their conductor but let's make this hard to get wrong by accident. This is why I hadn't added this as a feature before but in a test that needs to rely only on publish and not gossip, we have to be able to publish a little faster.